src/utils/hints/stateChannelsHints.js

```diff
@@ -0,0 +1,10 @@
+export const stateChannelsHints = {
+  stateChannelId: 'Unique identifier of the State Channel.',
+  status: 'State Channel can be Open or Closed. When status is Open the transactions lies outside of the blockchain network.',
+  participants: 'Initiator is an account that created the State Channel via ChannelCreateTx. Responder is an account that interacted with the State Channel as a last one.',
+  onChainUpdates: 'Amount of an off-chain transactions within the State Channel.',
+  locked: 'Currently deposited amount to the State Channel by it\'s participants.',
+  lastKnownRound: '',
```

Comment:
Here I was a bit desperate to coming up with a hint. In the sources I went through I haven/t find anything meaningful ./ Can we ignite the discussion in here, please?

Reply:
Thanks for the suggestion, I have used that.